src/internal/connector/sharepoint/data_collections.go

```diff
@@ -71,17 +80,53 @@ func DataCollections(
 				if err != nil {
 					return nil, support.WrapAndAppend(site, err, errs)
 				}
-
-				collections = append(collections, spcs...)
 			}
 
+			collections = append(collections, spcs...)
+
 			foldersComplete <- struct{}{}
 		}
 	}
 
 	return collections, errs
 }
 
+func collectLists(
+	ctx context.Context,
+	serv graph.Servicer,
+	tenantID, siteID string,
+	scope selectors.SharePointScope,
+	updater statusUpdater,
+	ctrlOpts control.Options,
+) ([]data.Collection, error) {
+	logger.Ctx(ctx).With("site", siteID).Debug("Creating SharePoint List Collections")
+
+	if scope.Matches(selectors.SharePointSite, siteID) {
+		spcs := make([]data.Collection, 0)
+
+		dir, err := path.Builder{}.Append(siteID).
+			ToDataLayerSharePointPath(
+				tenantID,
+				siteID,
+				path.ListsCategory,
+				false)
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to create collection path for site: %s", siteID)
+		}
+
+		collection := NewCollection(dir, serv, updater.UpdateStatus)
+		collection.AddJob(siteID)
+
+		fmt.Printf("Collection:\nPath: %v\nJobs: %v\n", collection.fullPath, collection.jobs)
```

Comment:
Should be logged or removed.

Reply:
Resolved here [513b5d7](https://github.com/alcionai/corso/pull/1788/commits/513b5d7eea414a21e3cd2e5a96dd987c9caae65a)